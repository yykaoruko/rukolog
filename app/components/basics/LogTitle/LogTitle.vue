<template>
  <h2 class="log-title">
    {{ title }}
  </h2>
</template>

<script lang="ts">
import Vue from 'vue';
export default Vue.extend({
  name: 'LogTitle',
  props: {
    title: {
      type: String,
      required: true,
    },
  },
});
</script>

<style lang="scss" scoped>
.log-title {
  color: inherit;
}
</style>
