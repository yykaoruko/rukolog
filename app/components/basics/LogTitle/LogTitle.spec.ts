import Vue from 'vue';
import { shallowMount, Wrapper } from '@vue/test-utils';
import LogTitle from './LogTitle.vue';

describe('LogTitle', () => {
  describe('mount', () => {
    let wrapper: Wrapper<Vue>;

    beforeEach(() => {
      wrapper = shallowMount(LogTitle, {
        propsData: {
          title: 'title',
        },
      });
    });

    it('mountable', () => {
      expect(wrapper.isVueInstance()).toBe(true);
    });

    it('snapshot', () => {
      expect(wrapper.element).toMatchSnapshot();
    });
  });
});
