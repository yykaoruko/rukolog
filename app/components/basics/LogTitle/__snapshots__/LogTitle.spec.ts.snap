// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogTitle mount snapshot 1`] = `
<h2
  class="log-title"
>
  
  title

</h2>
`;
